### PR TITLE
change all bower_components to node_modules

### PR DESCRIPTION
**Pull request by ericawright**
Status:  **Ready for Review**
Owner: @ericawright
Reviewers: @mikenikles @MikeKlemarewski @johnboxall  @fractaltheory @stewartyu 
## Changes
- change bower to npm
- change zepto to JQuery 
## Todos:
- [x] Engineer +1

### CODE REVIEW

**Comment by jansepar** [comment]
Niiiiiiiiiiice ⭐️

File: README.md [hunk 1]
```diff
@@ -8,21 +8,17 @@ A mobile-first content fly-in UI plugin.
...
 
-### Velocity
+### Zepto Support
 
-If you are using Zepto, you need to load `bower_components/mobify-velocity/velocity.js` (this file comes with a jQuery shim bundled directly in it). If you are using jQuery, you need to load `bower_components/velocity/jquery.velocity.js`.
-
-### jQuery Support
-
-Pikabu supports jQuery but is not actively developed for it. You should be able to use Pikabu directly with jQuery 2.0. While we don't actively support jQuery for Pikabu, we welcome any and all issues and PRs to help us make it work.
+Pikabu supports Zepto but is not actively developed for it. You should be able to use Pikabu directly with Zepto. While we don't actively support Zepto for Pikabu, we welcome any and all issues and PRs to help us make it work.
```
Comment (marlowpayne) [inline README.md]: There should probably be a note here about what's the last Zepto-compatible version of Pikabu that we've released.

File: README.md [hunk 2]
```diff
@@ -40,17 +36,20 @@ We highly recommend using Require.js with Pikabu. To use Require, you have to re
...
+        'airbnb': 'dist/effect/airbnb',
+        'drawer-left': 'dist/effect/drawer-left',
+        'drawer-right': 'dist/effect/drawer-right',
+        'plugin': 'node_modules/plugin/dist/plugin.min',
+        'shade': 'node_modules/shade/dist/shade',
+        'lockup': 'node_modules/lockup/dist/lockup',
+        'deckard': 'node_modules/deckard/dist/deckard.min',
+        'pikabu': 'dist/pikabu',
+        'FastClick': 'node_modules/fastclick/lib/fastclick',
         // There are a few more within the 'effect' folder
```
Comment (marlowpayne) [inline README.md]: We can probably either remove this comment entirely, or actually add all the effects to this example config.

File: README.md [hunk 3]
```diff
@@ -40,17 +36,20 @@ We highly recommend using Require.js with Pikabu. To use Require, you have to re
...
-        'shade': 'bower_components/shade/dist/shade.min',
-        'lockup': 'bower_components/lockup/dist/lockup.min',
-        'deckard': 'bower_components/deckard/dist/deckard.min',
-        'bouncefix': 'bower_components/bouncefix.js/dist/bouncefix.min',
-        'velocity': 'bower_components/mobify-velocity/velocity',
-        // or jQuery version 'velocity': 'bower_components/mobify-velocity/jquery.velocity',
-
-        'drawer-left': 'bower_components/pikabu/dist/effect/drawer-left',
-        'drawer-right': 'bower_components/pikabu/dist/effect/drawer-right'
+        'text': 'node_modules/requirejs-text/text',
```
Comment (marlowpayne) [inline README.md]: I think requirejs' text plugin installs under `node_modules/text/text` instead of `node_modules/requirejs-text/text`.

Comment (ericawright) [inline README.md]: yup, thanks

**Comment by marlowpayne** [comment]
LGTM :+1:

**Pull request merged**